taskana-adapter-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java

```diff
@@ -83,40 +89,73 @@ public void multiple_user_task_process_instances_started_in_camunda_via_rest_sho
     }
 
     @WithAccessId(
-        userName = "teamlead_1",
-        groupNames = {"admin"})
+            userName = "teamlead_1",
+            groupNames = {"admin"})
     @Test
-    public void task_with_variables_should_result_in_taskanaTask_with_those_variables_in_custom_attributes()
-        throws JSONException, InterruptedException, TaskNotFoundException, NotAuthorizedException {
+    public void task_with_primitive_variables_should_result_in_taskanaTask_with_those_variables_in_custom_attributes()
+            throws JSONException, InterruptedException{
+
         String variables = "\"variables\": {\"amount\": {\"value\":555, \"type\":\"long\"},\"item\": {\"value\": \"item-xyz\"}}";
         String processInstanceId = this.camundaProcessengineRequester
-            .startCamundaProcessAndReturnId("simple_user_task_process", variables);
+                .startCamundaProcessAndReturnId("simple_user_task_process", variables);
         List<String> camundaTaskIds = this.camundaProcessengineRequester
-            .getTaskIdsFromProcessInstanceId(processInstanceId);
+                .getTaskIdsFromProcessInstanceId(processInstanceId);
+
+        Thread.sleep((long) (this.adapterTaskPollingInterval * 1.2));
+
```

Comment:
The thread should sleep 1.2 times the polling interval

Reply:
Done